### PR TITLE
Update README.md to list Url instead of Path

### PR DESCRIPTION
**Pull request by KieranFrance**
A simple update to match the correct variable names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ php convert.php
 }
 {
     "state": "processed",
-    "previewPath": "output\/c0096728-3490-4f5f-96a8-0f20a5a1244c\/file\/index.html",
-    "downloadPath": "output\/c0096728-3490-4f5f-96a8-0f20a5a1244c\/file.zip"
+    "previewUrl": "output\/c0096728-3490-4f5f-96a8-0f20a5a1244c\/file\/index.html",
```
Comment (leonatherton) [inline README.md]:
```suggestion
    "previewUrl": "http://localhost:8080/buildvu-microservice/output/c0096728-3490-4f5f-96a8-0f20a5a1244c/file/index.html",
```

File: README.md [hunk 2]
```diff
@@ -123,8 +123,8 @@ php convert.php
...
+    "downloadUrl": "output\/c0096728-3490-4f5f-96a8-0f20a5a1244c\/file.zip"
```
Comment (leonatherton) [inline README.md]:
```suggestion
    "downloadUrl": "http://localhost:8080/buildvu-microservice/output/c0096728-3490-4f5f-96a8-0f20a5a1244c/file.zip"
```

File: README.md (hunk 1 above)
Comment (KieranFrance) [reply README.md]: Instead of localhost would it be better to update the ReadMe to use the address for the trial server in the examples and here?

**Comment by KieranFrance** [comment]
@leonatherton I have updated based on your suggestions.

**Pull request merged**